### PR TITLE
Factorization

### PR DESCRIPTION
**Pull request by andrewbowler**
If I messed up the request I will fix it for next time!!

### CODE REVIEW

**Comment by QuotableWater7** [comment]
@andrewbowler fyi there aren't any files on this branch to review 😢

**Comment by QuotableWater7** [comment]
looks like you pushed the code to master already. reminder that for pull requests you need the work on a different branch

**Comment by QuotableWater7** [comment]
```py
import math										# For sqrt()

def factorization(x):
	upper_bound = math.floor(math.sqrt(x)) + 1

	# some of these y's won't be primes, but who currrrs.
	# if they aren't prime, we know x % y won't be 0 since a factor of y also wasn't able to divide x
	for y in range(2, upper_bound):
		if (x % y == 0):
			print(y)
			return factorization(int(x / y))

	# if we get here, then x is the final factor
	print(x)

num = int(input('Input a number: '))
factorization(num)
```

**Comment by QuotableWater7** [comment]
Another tip:

Let's imagine instead of writing a silly challenge in a script, you were writing functions that would be used by anyone who needs prime factorization in their app for whatever reason. Printing the values from within the function wouldn't be super useful to them because they wouldn't be able to use the values for anything else. To make the function more useful, you could return an array with all the answers (which then someone could print if they wanted, but they could also run other math calculations on them).

Long story short, maybe try modifying your (or my) implementation so that the function eventually returns an array with all the correct numbers, and then the answer gets printed at the end of the script.
